core/src/core-plugin-definitions.ts

```diff
@@ -321,6 +321,13 @@ export interface CameraOptions {
    * The height of the saved image
    */
   height?: number;
+  /**
+   * Whether to preserve the aspect ratio of the image.
+   * If this flag is true, the width and height will be used as max values
+   * and the aspect ratio will be preserved.
+   * Default: false
+   */
+  preserveAspectRatio?: boolean;
```

Comment:
Aspect ratio is already respected if only height or width is passed, so this should make more clear that is for when both height and width are passed

Reply:
I'd suggest a name like `legacyPreserveAspectRatio` since this is more of a bug then a feature. That will make it more obvious that this will be removed in 3.0.

Reply:
So what I'm supposed to do now? 😅 Leave it like that? Update the comment? Rename the flag?

Reply:
The name itself probably isn't that important. Could just leave it as-is and clarify in the comment (that's probably what jcesarmobile ment).

Perhaps something like this?

```js
  /**
   * Whether to preserve the aspect ratio of the image.
   * If this flag is true, the width and height will be used as max values
   * and the aspect ratio will be preserved. This is only relevant when
   * both a width and height is passed, when only one of them are provided
   * the aspect ratio is always preserved.
   * 
   * A future major version will change this behavior to be default,
   * and may also remove the flag altogether.
   * Default: false
   */
```

Reply:
Thank you @sandstrom. I think your comment describes it very well. What do you think @dwieeb?

Reply:
Agreed, that's perfect.